sdk/src/client/api/block_builder/input_selection/automatic.rs

```diff
@@ -124,26 +124,26 @@ impl<'a> ClientBlockBuilder<'a> {
         // Then select inputs with outputs from addresses.
         let selected_transaction_data = 'input_selection: loop {
             // Get the addresses in the BIP path/index ~ path/index+20.
-            let addresses = self
-                .client
-                .get_addresses(
-                    self.secret_manager
-                        .ok_or(crate::client::Error::MissingParameter("secret manager"))?,
-                )
+            let opts = GetAddressesOptions::from_client(self.client)
+                .await?
+                .with_coin_type(self.coin_type)
```

Comment:
Just curious, where was this previously done?

Reply:
I'm not sure actually 😅

Reply:
@Thoralf-M dis fine?

Reply:
Yes, before it was actually missing the coin type here and just used the default